podman_compose.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python3
+#!/usr/bin/env python3
```

Comment:
Can go back to the hardcoded interpreter path, but I think it more solid to discover interpreter with `env` 
```suggestion
#! /usr/bin/python3
```